hooks/web-socket/constants.ts

```diff
@@ -0,0 +1,133 @@
+export enum Application {
+    REFINERY = 'REFINERY',
+    COGNITION = 'COGNITION'
+}
+
+export enum CurrentPage {
+    PROJECTS = 'PROJECTS',
+    LOOKUP_LISTS_OVERVIEW = 'LOOKUP_LISTS_OVERVIEW',
+    NEW_PROJECT = 'NEW_PROJECT',
+    PROJECT_OVERVIEW = 'PROJECT_OVERVIEW',
+    DATA_BROWSER = 'DATA_BROWSER',
+    LABELING = 'LABELING',
+    HEURISTICS = 'HEURISTICS',
+    ADMIN_PAGE = 'ADMIN_PAGE',
+    USERS = 'USERS',
+    UPLOAD_RECORDS = 'UPLOAD_RECORDS',
+    PROJECT_SETTINGS = "PROJECT_SETTINGS",
+    MODELS_DOWNLOAD = "MODELS_DOWNLOAD",
+    ATTRIBUTE_CALCULATION = "ATTRIBUTE_CALCULATION",
+    LOOKUP_LISTS_DETAILS = "LOOKUP_LISTS_DETAILS",
+    MODEL_CALLBACKS = "MODEL_CALLBACKS",
+    LABELING_FUNCTION = "LABELING_FUNCTION",
+    ACTIVE_LEARNING = "ACTIVE_LEARNING",
+    ZERO_SHOT = "ZERO_SHOT",
+    CROWD_LABELER = "CROWD_LABELER",
+    RECORD_IDE = "RECORD_IDE",
+    EDIT_RECORDS = "EDIT_RECORDS",
+    NOTIFICATION_CENTER = "NOTIFICATION_CENTER",
+    EXPORT = "EXPORT",
+    COMMENTS = "COMMENTS",
+    BRICKS_INTEGRATOR = "BRICKS_INTEGRATOR",
+    CONFIG = "CONFIG",
+    CHAT_UI = "CHAT_UI",
```

Comment:
I think we should prefix none refinery pages e.g. COGNTION_CHAT_UI would help with understanding on where to look for it. Otherwise we could also add a dict strucutre with the correct combination as REFINERY as app & CHAT_UI as page shouldn't be used by a dev

Reply:
tbough it would most likely be noticed anyway as the dict access would fail :)

Reply:
Added the prefix, and added the check for using a valid combination